recipes-core/customizations-example/files/terminal_resize.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+#
+# Copyright (c) Siemens AG, 2022
+#
+# Authors:
+#  Chao Zeng <chao.zeng@siemens.com>
+#
+# SPDX-License-Identifier: MIT
+
+terminal_resize() {
+    local IFS='[;' escape screen_size cols rows
+    # detect the size of the screen
+    echo -ne '\e7\e[r\e[999;999H\e[6n\e8'
+    read -t 5 -sd R escape screen_size || {
+        echo unsupported terminal emulator. >&2
+        return 1
```

Comment:
This concerns me: What if the user has different emulator, and this path is taken. Will the login session abort?

Reply:
No, just the simulation directly returns to 1 to see what happens

Reply:
Also `unsupported terminal emulator.` is too vague, unsupported for what? If only for resize, then this would be too wide warning, seems to me like 'my terminal emulator is not supported at all on iot2050'.

Reply:
This is so clear to show the `terminal emulator` can not be supported for auto-size.
And this is error log and if there is some wrong we can also get the info from this output

Reply:
I mean that, just show the error message that the current in-use term emu does not support resizing.

Reply:
...and then better do not bail out with an error code to avoid any surprises. I didn't find a specification yet regarding how shells should react on non-zero exit codes, but better safe than sorry.